web/styles/settings.css

```diff
@@ -1601,12 +1610,12 @@ label.preferences-radio-choice-label {
 
 #edit-user-form {
     .person_picker {
-        min-width: 206px;
+        min-width: var(--modal-input-width);
     }
 
     & textarea {
-        width: max(206px, 25vw);
-        max-width: 320px;
+        width: max(var(--modal-input-width), 25vw);
+        max-width: var(--modal-input-width);
```

Comment:
Textarea's width is also different from others. 
![image](https://github.com/user-attachments/assets/29add336-db68-4a2a-83f0-33270a59f4c9)

Reply:
Thanks, fixed!

```diff
     & textarea {
-        width: calc(var(--modal-input-width) - 6px);
+        /* Subtract 6px of padding from side */
+        width: calc(var(--modal-input-width) - 12px);
     }
 }
 
```